spellcheck/platform/linux/spellcheck_linux.cpp

```diff
@@ -33,35 +33,22 @@ class EnchantSpellChecker {
 	EnchantSpellChecker(const EnchantSpellChecker&) = delete;
 	EnchantSpellChecker& operator =(const EnchantSpellChecker&) = delete;
 
+	using DictPtr = std::unique_ptr<enchant::Dict>;
+
 	std::unique_ptr<enchant::Broker> _brokerHandle;
-	std::vector<std::unique_ptr<enchant::Dict>> _validators;
+	std::vector<DictPtr> _validators;
```

Comment:
Here you actually have the vector that contains no more than one validator.

Reply:
_temporary_
Does it make sense to remove the vector when this is a temporary solution?